hack/cluster-deploy.sh

```diff
@@ -52,7 +52,10 @@ EOF
 _kubectl apply -f ${MANIFESTS_OUT_DIR}/release/kubevirt-operator.yaml
 
 if [[ "$KUBEVIRT_PROVIDER" =~ os-* ]] || [[ "$KUBEVIRT_PROVIDER" =~ okd-* ]]; then
-    _kubectl create -f ${MANIFESTS_OUT_DIR}/testing/ocp
+
+    if [ -d "${MANIFESTS_OUT_DIR}/testing/ocp" ]; then
```

Comment:
Was this the only file in ocp? If yes, then we could just remove the whole if statement.

Reply:
@rmohr yes it was, I wanted to keep this in case we need it later on. But I'm fine with removing it. Should I remove all logic for ocp specific manifests then? There is also something in tests/utils.go, which checks for manifests ending in for-ocp.yaml. Wdyt?

Reply:
cleaned up everything. Also removed another statement about adding the operator to the prvileged SCC, which is not needed because there are RBAC rules for it.